submissions/dashakim/js-dom/index.css

```diff
@@ -0,0 +1,91 @@
+* {
+    box-sizing: border-box;
+}
+
+html,
+body {
+    background-color: rgb(235, 235, 235);
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    color: rgb(37, 37, 37);
+    margin: 0 auto;
+    text-align: center;
+}
+
+ul,
+li {
+    list-style: none;
+}
+
+li:hover {
+    background-color: rgba(136, 133, 133, 0.103);
+    cursor: pointer;
+
+}
+
+li:active {
+    background-color: rgba(228, 228, 228, 0.288);
+}
+
+.book-list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+    margin: 0;
+    padding: 0
+}
+
+.book-item {
+    border: 1px solid rgb(255, 255, 255);
+    padding: 19px;
+    /* margin: 10px; */
+
+}
+
+.container {
+    max-width: 920px;
+    margin: 0 auto;
+
+
+}
+
+h1 {
+    padding: 20px 0 0 0;
+
+}
+
+.content {
+    text-align: center;
+    background-color: #fff;
+    padding: 10px 0 40px 0;
+}
+
+.content p {
+    float: left;
+    max-width: 520px;
+    text-align: justify;
+    margin-left: 50px;
+    padding: 0;
+    white-space: pre-wrap
+}
+
+.active {
+    background-color: #fff;
+}
+
+@media(max-width:512px) {
+    .book-list {
+        flex-direction: column;
+        justify-content: space-between;
+    }
+
+    .content p {
+        margin: 0;
+    }
+}
+
+@media(max-width:920px) {
+    .content p {
+        margin-left: 80px;
+    }
+}
```

Comment:
Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).